### PR TITLE
sync_card: country may be null for "Diners Club" cards

### PR DESCRIPTION
**Pull request by bessiash**
country field may be null for "Diners Club" cards (for example "30569309025904" or "38520000023237" in test mode). without this changes we have error "null value in column "country" violates not-null constraint".

### CODE REVIEW

**Comment by ossanna16** [comment]
Thanks for the PR @bessiash! Can this be merged @paltman?

**Comment by bessiash** [comment]
I'm glad to be helpful @ossanna16

**Comment by paltman** [comment]
@bessiash can you add a test to provide that an empty string will work in creating a card.  I am pretty sure you need to add `blank=True` (and create the subsequent migration that Django will require even though it's not a DDL change) to the `Card` model.

**Comment by paltman** [comment]
@bessiash would love to merge this but really want a test to validate that they change works.  I suspect you need to add `blank=True` to the `country` field on `Card`.

**Comment by bessiash** [comment]
@paltman yeah it's work fine. check f2a8acde69db76658ef45da991563c021fd24c25 for review. sorry for long time of response.

File: pinax/stripe/actions/sources.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def sync_card(customer, source):
         address_zip=source["address_zip"] or "",
         address_zip_check=source["address_zip_check"] or "",
         brand=source["brand"],
-        country=source["country"] or "",
+        country=source["country"],
```
Comment (bessiash) [inline pinax/stripe/actions/sources.py]: @paltman I think I misunderstood your previous comment. sorry for it.

Comment (bessiash) [inline pinax/stripe/actions/sources.py]: @paltman as you asked me I had added test to validate None country value. check 7eac4a887a8cbdd2f555f6eb05e3588df7258fe2 for it.

also I had to redo migration in order to avoid conflict with other migration after merge with master (35416e8862a073337f06efe72a53d38a9100e8f0).

**Comment by paltman** [comment]
@bessiash all looks great but what is the binary file that just got added called `stripe`?

**Comment by bessiash** [comment]
oh... sorry @paltman. looks like I accidentally added test db file to commit.  give me a sec to remove it.

**Comment by paltman** [comment]
@bessiash thanks!
